android/src/main/java/com/swmansion/reanimated/layoutReanimation/AnimationsManager.java

```diff
@@ -505,8 +537,36 @@ private ViewManager resolveViewManager(int tag) {
     try {
       return mReanimatedNativeHierarchyManager.resolveViewManager(tag);
     } catch (Exception e) {
-      e.printStackTrace();
+      //      e.printStackTrace();
       return null;
     }
   }
+
+  private Point convertPoint(Point fromPoint, View parentView) {
```

Comment:
how does it convert the point? Maybe rename to sth that'd give better intuition about that, for example `convertRelativePointToScreenLocation`